### DISCUSSION TITLE
Cannot get web GUI to load.  fresh install.

### DISCUSSION THREAD

**Discussion by JabbaDaHutt762** [RandomNinjaAtk/docker-lidarr-extended#109] [Q&A] [answered]
Any assistance would be appreciated.  Running Unraid 6.11.1.  Everything is up to date.  

![lidarr docker setup](https://user-images.githubusercontent.com/57692223/197223570-23be4da9-a2c1-4574-a9b3-71002675bada.PNG)
![lidarr port](https://user-images.githubusercontent.com/57692223/197223595-f6c3392d-a9b3-4956-96ef-e5462cc7b574.PNG)
![arrs](https://user-images.githubusercontent.com/57692223/197223608-6811e9da-2bbb-4309-8239-6b80db976bb1.PNG)
![nord ports](https://user-images.githubusercontent.com/57692223/197223626-817bc841-41c5-4ceb-adcc-0bbbc350e47a.PNG)

**Comment by RandomNinjaAtk** [comment, marked as answer]
Your issue is your likely caused by your custom networking configuration because your piping it through a VPN, I cannot help you, but others may be able to assist. If your remove your custom networking, I bet it works....

**Reply by JabbaDaHutt762** [reply]
i got it!  the vpn wasnt the issue it was the location of the log argument.

**Reply by JabbaDaHutt762** [reply]
I had the log argument in the highlighted section.  as soon as i removed it, worked like a champ.  
![log argument](https://user-images.githubusercontent.com/57692223/197236680-0fd1f8cf-f285-4826-bdbd-00fd1f04cebf.PNG)

**Reply by JabbaDaHutt762** [reply]
what is the correct separator for additional extra parameters if i want to add the log back in?

**Reply by RandomNinjaAtk** [reply]
I believe its just a single space (which is what I've always used), you typically don't use Post Arguments.

**Reply by JabbaDaHutt762** [reply]
Thanks man.  This looks great.  Where is the section for music videos?  is it mixed in with the standard library?

**Reply by RandomNinjaAtk** [reply]
Review the various log files in lidarr, everything is via script. So you have to check the logs to see what is going on. Because everything is scripted and runs in the background, it kinda works/feels like magic...

Videos will download to the specified music-videos volume location.

**Reply by JabbaDaHutt762** [reply]
ok so there is no visual aid for the music video collection similar to sonarr or radarr?

**Reply by RandomNinjaAtk** [reply]
correct

**Reply by JabbaDaHutt762** [reply]
bro another tab in there with videos would be stellar but i was just watching the logs and you are a genius.  this is pretty sick.
